content/management/configuration-api/v3.3/index.mdx

```diff
@@ -3497,6 +3497,7 @@ Informs that a user has seen events up to a specific time.
 | `filters.chat_member_ids`                | no       | `object`   | Only one filter (`agents_any` or `agents_exclude`) is allowed.                                                                                            |
 | `filters.chat_member_ids.agents_any`     | no       | `[]string` | Array of Agents' ids; if any specified Agent is in the chat, the webhook will be triggered.                                                               |
 | `filters.chat_member_ids.agents_exclude` | no       | `[]string` | Array of Agents' ids; If any specified Agent is in the chat, the webhook will not be triggered.                                                           |
+| `owner_client_id`                        | no       | `string`   | ClientID of desired webhook owner. Required when Personal Access Token is used to authorize.                                                              |
```

Comment:
```suggestion
| `owner_client_id`                        | no       | `string`   | The Client ID for which the webhook will be registered. It's required when a Personal Access Token is used for authorization.                                                              |
```

Reply:
I think we need to add the method definition for **[Register Webhook](https://github.com/livechat/livechat-public-docs/pull/814/files#diff-8ad53a5eba0e6c361340079931ab39577b913fa57076b7150d8d07cb95da1fb2R3478)**. I propose:

> Registers a webhook for the Client Id (application) associated with the access token used in the request. Later on, registered webhooks can be [enabled](#enable-webhook) for a given license.